JavaScript/2-const-let.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
-console.dir({ bubbleEffect });
```

Comment:
```suggestion
console.dir({ hoisted }); // { hoisted: undefined }
```
We need this to show variable value before assignment